tests/demo_inheritance/conflicted/child/copier.yml

```
@@ -0,0 +1 @@
+_extends: ./tests/demo_inheritance/conflicted/parent
```

If this works as it is written here, I'd consider this a bug.

You cannot predict from which folder the user will launch copier. Instead, all paths should always be relative to the very `copier.yml` file that defines them.

So, in this case, this should be:

```suggestion
_extends: ../parent
```

... because from this `copier.yml` file, the parent folder is found there.

Also have you considered the possibility of extending several at once? We could define this as a list:

```suggestion
_extends:
  - ../parent
```

The same applies for other test cases here.

---

@Yajo Thx. I will investigate this. Please consider opening a short issue if you think there is indeed a bug or potential improvement here, so we can properly track things.

---

Well, this is WIP, I don't think I need another issue, isn't it?